src/components/hooks/UseToast/index.tsx

```diff
@@ -0,0 +1,186 @@
+"use client";
+
+import * as React from "react";
+
+import type { ToastActionElement, ToastProps } from "@/components/ui/Toast";
+
+const TOAST_LIMIT = 1;
+const TOAST_REMOVE_DELAY = 1000000;
+
+type ToasterToast = ToastProps & {
+  id: string;
+  title?: React.ReactNode;
+  description?: React.ReactNode;
+  action?: ToastActionElement;
+};
+
+let count = 0;
+
+function genId() {
+  count = (count + 1) % Number.MAX_SAFE_INTEGER;
+  return count.toString();
+}
+
+type ActionType = {
```

Comment:
imo enum will be more suitable for the type

Reply:
Okay kak, lemme try to adjust it

Reply:
Done kak @eezhal92